### ISSUE TITLE
Translating ASN1::BER values to plaint text

### ISSUE THREAD

**Issue by lebogan** [aca-labs/crystal-snmp#2]
I stumbled on your excellant shard and have used it to replace the snmpget utility from net-snmp. I have a couple of questions:

1. Is there a way to use oid names, i.e., sysName without resorting to snmptranslate?
2. How do you print out the results of client.walk(oid) in plaintext like snmpwalk?
```
#<ASN1::BER:0x7fb57cc2c7e0 tag_class=Universal constructed=false tag=OctetString tag_number=4 extended=false size=34 payload=Bytes[84, 104, 101, 32, 77, 73, 66, 32, 109, 111, 100, 117, 108, 101, 32, 102, 111, 114, 32, 83, 78, 77, 80, 118, 50, 32, 101, 110, 116, 105, 116, 105, 101, 115]>
"1.3.6.1.2.1.1.9.1.3.
```
Thank you for all your hard work on this.

**Comment by stakach** [comment]
1. would involve creating a lookup for all the known OIDs or implementing a MIB reader - happy to accept pull requests but not so important for my use cases.

2. is not too hard but I won't have time to implement it for a couple of months.

The inspect function is here: https://github.com/spider-gazelle/bindata/blob/master/src/bindata/asn1.cr#L145

We could add a case statement for the various `Universal` tags and have the function print a human readable value.
I think we'd have to be careful to catch any errors and fallback to printing the bytes as data structures like RSA keys are stored as ASN.1 BER-encoded Integers however the keys are a huge number of bytes long and the integer reader I implemented only works for integers up to 64bits long (as I couldn't work out how to de-serialize into crystals BigInt)

There are also some universal SNMP types too that are not part of the ASN.1 standard. I've implemented a few of these (but many more to do) - so another option would be to build an inspect function in this class:
https://github.com/spider-gazelle/crystal-snmp/blob/master/src/snmp/varbind.cr
I also want to write more helpers for decoding common data structures: https://github.com/spider-gazelle/crystal-snmp/blob/master/src/snmp/helpers/if_entry.cr

My main use case is interrogating switches.
